tutorials/classical_simulation/Simulating with SPAM errors.ipynb

```diff
@@ -0,0 +1,335 @@
+{
```

Comment:
<p>I am not sure to get <code>since the randomness comes from state preparation alone.</code>. I would have thought  " since what makes a simulation noisy is its state preparation, and not its measurement."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>I agree it could be worded in a better way. What it means is that simulating state preparation errors requires you to sample a probability distribution to obtain the atoms that were badly prepared, while measurement errors can be applied coherently to a state vector or density matrix.</p>

Reply:
<p>Ok your explanation is definitely better, thanks !</p>